doc/modules/changes/20200701_haoyuanli

```diff
@@ -0,0 +1,8 @@
+New: Added function phase_average_value, namespace PhaseUtilities to namespace MaterialModel::MaterialUtilities.
+This new function averages parameters(e.g density, viscosity) across all phases for a composition. It
+takes parameters of phases and values of gamma functions as inputs, applys an arithmetic or 
+logarithmic averaging and return the averaged value. A enum type PhaseUtilities::PhaseAveragingOperation 
+is defined to determine the type of operation(e.g arithmetic, logarithmic) to perform in the phase_average_value
+function.
```

Comment:
Thanks for the changelog. We usually try to keep the description here as short as possible, the exact usage of the function can be seen in the documentation of the header file. This is more a note to users that this new functionality exists. Here is how I would change it:
```suggestion
New: Added a new function PhaseUtilities::phase_average_value to the namespace MaterialModel::MaterialUtilities.
This new function averages parameters(e.g density, viscosity) across all phases for a composition.
```